client/logs_test.go

```diff
@@ -111,6 +116,38 @@ func (cs *clientSuite) TestLogsAll(c *check.C) {
 `[1:])
 }
 
+func (cs *clientSuite) TestLogsLong(c *check.C) {
+	shortLog := `{"time":"2021-05-03T03:55:49.360994155Z","service":"thing","message":"log 1\n"}`
+	var longLog strings.Builder
+	longLog.WriteString(`{"time":"2021-05-03T03:55:49.460994155Z","service":"verbose","message":"`)
+	for i := 0; i < maxMessageSize; i++ {
+		longLog.WriteString("a")
+	}
+	longLog.WriteString(`\n"}`)
+	logs := []string{shortLog, longLog.String(), ""}
+	cs.rsp = strings.Join(logs, "\n")
```

Comment:
I was confused at first why there was an empty string in the slice (to ensure there's a trailing newline). Would just `cs.rsp = shortLog + "\n" + longLog + "\n"` be clearer?

Could we also ensure that the next log *after* the long log comes through correctly? So maybe `shortLog + longLog + shortLog` type of thing.